### PR TITLE
[wip] Release 2.9.2

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**
:)

### CODE REVIEW

File: doc/en/example/pythoncollection.rst [hunk 1]
```diff
@@ -180,7 +180,7 @@ and a setup.py dummy file like this::
 then a pytest run on Python2 will find the one test and will leave out the
 setup.py file::
 
-    $ py.test --collect-only
+    #$ py.test --collect-only
```
Comment (The-Compiler) [inline doc/en/example/pythoncollection.rst]: Is this intentional?

Comment (RonnyPfannschmidt) [inline doc/en/example/pythoncollection.rst]: yes, regendoc woulduse python 3.5 due to non-multi-version see #1573

**Comment by nicoddemus** [comment]
Thanks for tackling this! :smile:

**Comment by RonnyPfannschmidt** [comment]
don't praise too early, i got sidetracked, will finish the changes now and test on windows/devpi

**Comment by nicoddemus** [comment]
Feel free to fork [devpi-cloud-tester](https://github.com/nicoddemus/devpi-cloud-tester) to test your devpi package. You have to enable AppVeyor and Travis for your forke and edit `appveyor.yml` and `.travis.yml` to point to your devpi channel.

File: doc/en/announce/index.rst [hunk 2]
```diff
@@ -7,6 +7,8 @@ Release announcements
 
 
    sprint2016
+   release-2.9.1
+   release-2.9.1
```
Comment (The-Compiler) [inline doc/en/announce/index.rst]: I guess it's not intended that this is listed twice, right?

**Pull request merged**